### PR TITLE
docs: correct middleware file location

### PR DESCRIPTION
**Pull request by chrisgreen1993**
## ☕️ Reasoning

Updates the docs to use the correct location of the middleware file.

## 🧢 Checklist

- [x] Documentation
- [ ] Tests
- [x] Ready to be merged

## 🎫 Affected issues

### CODE REVIEW

**Review by ndom91 — COMMENTED** [review]
Tbh I feel like the change is more confusing..

File: docs/docs/tutorials/securing-pages-and-api-routes.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default function Page() {
 
 ### Next.js (Middleware)
 
-With NextAuth.js 4.2.0 and Next.js 12, you can now protect your pages via the middleware pattern more easily. If you would like to protect all pages, you can create a `middleware.js` file in your root `pages` directory which looks like this:
+With NextAuth.js 4.2.0 and Next.js 12, you can now protect your pages via the middleware pattern more easily. If you would like to protect all pages, you can create a `middleware.js` file in at the root or in the `src` directory (same level as your `pages`) which looks like this:
```
Comment (ndom91) [inline docs/docs/tutorials/securing-pages-and-api-routes.md:45]:
```suggestion
With NextAuth.js 4.2.0 and Next.js 12, you can now protect your pages via the middleware pattern more easily. If you would like to protect all pages, you can create a `middleware.js` file in the root `pages` directory or in the `src` directory at the same level as `pages`,  which looks like this:
```

**Review by ndom91 — COMMENTED** [review]
Maybe something more like this?

File: docs/docs/tutorials/securing-pages-and-api-routes.md (hunk 1 above)
Comment (chrisgreen1993) [reply docs/docs/tutorials/securing-pages-and-api-routes.md:45]: I just used the same language as the nextjs middleware docs here: https://nextjs.org/docs/advanced-features/middleware#using-middleware as I was originally putting the file in the `pages` directory and it was having no effect.

**Comment by balazsorban44** [comment]
Updated in https://github.com/nextauthjs/next-auth/pull/5981/files for the `v4` branch. The authjs.dev version will get a complete rewrite anyway, so I think this is safe to close, but thanks anyway!

**Pull request closed without merging**